pyiron_base/jobs/job/extension/server/generic.py

```diff
@@ -182,15 +182,15 @@ def queue(self, new_scheduler):
                     run_time_max=self.run_time,
                     memory_max=self.memory_limit,
                 )
-                if cores != self.cores:
+                if self.cores is not None and cores != self.cores:
```

Comment:
If either `self.{cores,run_time,memory_limit}` is `None` then the corresponding branches are not taken and the value cannot be updated, right?  Is that what we want?

Reply:
But if I do `job.server.cores = ...` in the notebook, this has be reflected, no?

Reply:
Yes, in that case `self.cores` would be set and the route would be taken, wouldn't it? The `cores` we compare `self.cores` with comes just from the `queue_adapter.check_queue_parameters()` which was fed by `self.cores` (at least in the case we just change the scheduler, i.e. the queue).

Reply:
Ah, sorry, I had misread the code. This is all in the `queue` setter and the other setters are not changed.  A-ok for me then.